src/main/java/de/fraunhofer/aisec/cpg/graph/Node.java

```diff
@@ -116,7 +120,7 @@
 
   /** List of annotations associated with that node. */
   @SubGraph("AST")
-  protected List<Annotation> annotations;
+  public List<Annotation> annotations = new ArrayList<>();
```

Comment:
Why is this member public?

Reply:
All members are public to allow reflection access from outside of the module without having to use `--add-opens` workarounds in Java >= 9.

Reply:
All the other members are protected?

Reply:
Err... Anyway, it's now consistent with the other fields :-)